crates/types/src/config/networking.rs

```diff
@@ -72,6 +72,20 @@ pub struct NetworkingOptions {
     /// If network latency is high, it's recommended to set this to a higher value.
     /// Maximum theoretical value is 2^31-1 (2 GiB - 1), but we will sanitize this value to 500 MiB.
     data_stream_window_size: NonZeroByteCount,
+
+    /// # Max Encoding Message Size
+    ///
+    /// Limits the maximum size of a encoded message.
+    ///
+    /// Default: `10MB`
+    pub max_encoding_message_size: NonZeroByteCount,
+
+    /// # Max Decoding Message Size
+    ///
+    /// Limits the maximum size of a decoded message.
+    ///
+    /// Default: `10MB`
+    pub max_decoding_message_size: NonZeroByteCount,
```

Comment:
What makes this a little bit confusing is that the actual networking grpc client and server we are using for our internal networking (for which these options are designed), does not use these values and instead uses a hard coded 32 MB limit.

Reply:
The network fabric indeed uses a hard coded 32MB limit. But those configuration as you already know are used to the exposed gRPC services (node grpc, metadata-server, metadata-proxy, etc...). 

I will try to document this clearly